### PR TITLE
Export-to-csv unit test cases using @oclif/test

### PR DESCRIPTION
**Pull request by aman19K**
- [x] unit test cases using @oclif/test

### CODE REVIEW

File: packages/contentstack-export-to-csv/test/unit/commands/export-to-csv.test.js [hunk 1]
```diff
@@ -1,219 +1,289 @@
...
+      .it('CSV file should be created');
   });
 });
 
-describe('Export to csv command with action = users', () => {
-  let inquireStub;
-  let errorStub;
-  let consoleLogStub;
+describe('export-to-csv with action entries', () => {
+  if (!fs.existsSync(directory)) fs.mkdirSync(directory);
```
Comment (antonyagustine) [inline packages/contentstack-export-to-csv/test/unit/commands/export-to-csv.test.js:129]: Why are we creating a folder in the unit test case.?

File: packages/contentstack-export-to-csv/test/unit/commands/export-to-csv.test.js [hunk 2]
```diff
@@ -1,219 +1,289 @@
...
-    assert.calledOnce(inquireStub);
-  });
-
-  it('Should ask for org when org is not passed', async () => {
-    const args = ['--action', 'entries'];
-    await ExportToCsvCommand.run(args);
-    assert.calledOnce(inquireStub);
-  });
+describe('export-to-csv with action taxonomies', () => {
+  if (!fs.existsSync(directory)) fs.mkdirSync(directory);
```
Comment (antonyagustine) [inline packages/contentstack-export-to-csv/test/unit/commands/export-to-csv.test.js:16]: Why are we creating a folder in the unit test case.?

**Review by abhinav-from-contentstack — APPROVED** [review]

File: packages/contentstack-export-to-csv/test/unit/commands/export-to-csv.test.js (hunk 2 above)
Comment (aman19K) [reply packages/contentstack-export-to-csv/test/unit/commands/export-to-csv.test.js]: @antonyagustine removed it.

File: packages/contentstack-export-to-csv/test/unit/commands/export-to-csv.test.js (hunk 1 above)
Comment (aman19K) [reply packages/contentstack-export-to-csv/test/unit/commands/export-to-csv.test.js]: @antonyagustine removed

File: packages/contentstack-export-to-csv/test/unit/commands/export-to-csv.test.js [hunk 3]
```diff
@@ -1,219 +1,293 @@
...
+      .stub(process, 'chdir', () => {})
+      .nock(cma, (api) => {
+        api.get('/v3/user?include_orgs_roles=true').reply(200, { user: mockData.users[0] });
+      })
+      .nock(cma, (api) => {
+        api.get(`/v3/organizations/${mockData.organizations[0].uid}/roles`).reply(200, { roles: mockData.roles });
+      })
+      .nock(cma, (api) => {
+        api.get('/v3/user?include_orgs_roles=true').reply(200, { user: mockData.users[0] });
+      })
```
Comment (antonyagustine) [inline packages/contentstack-export-to-csv/test/unit/commands/export-to-csv.test.js:253]: Both user APIs look identical. Can we add `.persist()` and have only one nock method.?

File: packages/contentstack-export-to-csv/test/unit/commands/export-to-csv.test.js [hunk 4]
```diff
@@ -1,219 +1,293 @@
...
+      .nock(cma, (api) => {
+        api.get(`/v3/organizations/${mockData.organizations[0].uid}/roles`).reply(200, { roles: mockData.roles });
+      })
+      .nock(cma, (api) => {
+        api.get('/v3/user?include_orgs_roles=true').reply(200, { user: mockData.users[0] });
+      })
```
Comment (antonyagustine) [inline packages/contentstack-export-to-csv/test/unit/commands/export-to-csv.test.js:284]: Both user APIs look identical. Can we add `.persist()` and have only one nock method.?

File: packages/contentstack-export-to-csv/test/unit/commands/export-to-csv.test.js (hunk 3 above)
Comment (aman19K) [reply packages/contentstack-export-to-csv/test/unit/commands/export-to-csv.test.js]: done

File: packages/contentstack-export-to-csv/test/unit/commands/export-to-csv.test.js (hunk 4 above)
Comment (aman19K) [reply packages/contentstack-export-to-csv/test/unit/commands/export-to-csv.test.js]: done

**Review by antonyagustine — APPROVED** [review]

**Pull request merged**